utils/volumeutils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	config "github.com/spf13/viper"
+)
+
+func GetVolumeDir(volumeName string) string {
```

Comment:
I'm wondering if these two functions could be associated with the `Volinfo` and `Brickinfo` types themselves.